pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -821,6 +826,77 @@ func (r *ReconcileHyperConverged) ensureKubeVirtTemplateValidator(instance *hcov
 	return r.client.Status().Update(context.TODO(), instance)
 }
 
+func newKubeVirtStorageConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *corev1.ConfigMap {
+	var volumeMode string
+
+	if bareMetalPlatform {
+		volumeMode = "Block"
+	} else {
+		volumeMode = "Filesystem"
+	}
+
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-config-storage-class-defaults",
+			Labels:    labels,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			"accessMode": "ReadWriteMany",
```

Comment:
With this set, no VM created by UI (Wizard) can be started when local SC is used.